event_model/schemas/event_descriptor.json

```diff
@@ -27,6 +27,13 @@
                         },
                         "description": "The shape of the data.  Null and empty list mean scalar data."
                     },
+                    "dims": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        },
+                        "description": "The names for dimensions of the data. Null and empty list if scalar data"
```

Comment:
I think "Null _or_ empty list if scalar data" would make more sense than "and" here.